src/graphql/entries.ts

```diff
@@ -30,21 +30,26 @@ export async function updateEntry({
   updateFromEntryEditor,
   locale,
   entityReferenceMap,
+  aliases
 }: UpdateEntryProps): Promise<Entity & { sys: SysProps }> {
   if (dataFromPreviewApp.sys.id !== updateFromEntryEditor.sys.id) {
     return dataFromPreviewApp;
   }
 
-  const copyOfDataFromPreviewApp = { ...dataFromPreviewApp };
+  const copyOfDataFromPreviewApp = clone(dataFromPreviewApp);
   const { fields } = contentType;
 
+
+  console.log(">> dataFromPreviewApp", dataFromPreviewApp)
+  console.log(">> aliases", aliases)
+
   logUnrecognizedFields(
-    fields.map((f) => f.apiName ?? f.name),
+    fields.map((field) => getAliasForFieldName(field.apiName ?? field.name)),
```

Comment:
I don't like that we have to do it twice, maybe we can include the log into the for loop